mmf/utils/distributed.py

```diff
@@ -31,8 +35,13 @@ def synchronize():
 
     dist.barrier()
 
+def is_xla():
+    global _USE_XLA
```

Comment:
you can remove this line. `global blabla` is only needed if you're editing a global variable.